custom_components/tuya_local/devices/casdon_td_pro_2.yaml

```diff
@@ -0,0 +1,206 @@
+name: Casdon TD Pro 2
+products:
+  - id: wab6oustdukujpkn
+    name: 蒸烤箱TD-pro二代
```

Comment:
If this is sold in China market only, then a Chinese name is OK here, but if it is sold outside China also, it may be better as "Casdon TD Pro 2" so the user does not need to read Chinese to recognise it.

Reply:
I searched for product information, and found that this company only sells its products in China, but I feel using English would create a more harmonious feel.